src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -134,3 +149,21 @@ $table-cell-height-half: $table-cell-height / 2;
   color: $foreground-2;
   cursor: default;
 }
+
+.kd-replicasetdetail-no-events {
+  margin-top: 4 * $baseline-grid;
+  text-align: center;
+}
+
+md-icon {
+  &.kd-replicasetdetail-no-events-icon {
+    color: $muted;
+    font-size: 1.5 * $headline-font-size-base;
```

Comment:
I'm not sure. Without this icon seems really small compared to the text.

![screenshot from 2016-01-26 15-03-14](https://cloud.githubusercontent.com/assets/2285385/12582530/081ff224-c43e-11e5-9d75-7e8218c7bbf7.png)

vs

![screenshot from 2016-01-26 15-03-25](https://cloud.githubusercontent.com/assets/2285385/12582534/0b4c4f10-c43e-11e5-9a0f-12a4313ca284.png)

What do you think?